### PR TITLE
update studentlist

### PR DESCRIPTION
**Pull request by Artanic30**

### CODE REVIEW

File: src/instructor/components/CourseList/Add.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    var check = (rule, value, callback) => {
+      if (!value) {
+        return callback(new Error('不能为空'))
+      }
+      setTimeout(() => {
+        callback()
+      }, 1000)
+    }
+    return {
+      ruleForm2: {
```
Comment (nichujie) [inline src/instructor/components/CourseList/Add.vue]: Try better variable name. Or if there is no second form, simply write 'form'

File: src/instructor/components/AddAssignment/Index.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
         </el-col>
         <el-col :span="18" style="float: right;">
             <v-state v-if="change" v-on:changeState="changeState"></v-state>
-            <v-add v-if="!change" v-on:goBack="goBack" style="margin-right: 15%"></v-add>
+            <v-add v-else v-on:goBack="goBack" style="margin-right: 15%"></v-add>
```
Comment (nichujie) [inline src/instructor/components/AddAssignment/Index.vue]: Tags or attributes names starting with 'v' indicates that they belong to Vue.js. Your custom element should have a name to distinguish from HTML native elements. (e.g. do not use <aside> or <v-aside>, use <assignment-aside> or <qlt-aside>)

Comment (nichujie) [reply src/instructor/components/AddAssignment/Index.vue]: do not use 'aside' (HTML5 native tag) or 'v-aside', try 'assignment-aside' or 'qlt-aside'

File: src/instructor/components/CourseList/Add.vue [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      const loading = this.$loading({
+        lock: true,
+        text: 'Loading',
+        spinner: 'el-icon-loading',
+        background: 'rgba(0, 0, 0, 0.7)'
+      })
+      setTimeout(() => {
+        loading.close()
+        this.$emit('goBack')
+      }, 500)
```
Comment (nichujie) [inline src/instructor/components/CourseList/Add.vue]: So this is some kinda like a cheating loading animation?

File: src/student/components/ScoreBoard/Aside.vue [hunk 4]
```diff
@@ -26,7 +26,7 @@
       </el-col>
       <el-col :span="15">
         <el-breadcrumb>
-          <el-breadcrumb-item :to="{ path: '/course' }" style="font-size: 25px;color: black!important">Course</el-breadcrumb-item>
+          <el-breadcrumb-item :to="{ path: '/home/course/'+ this.coInfo.uid }" style="font-size: 25px;color: black!important">Course</el-breadcrumb-item>
```
Comment (alienmao) [inline src/student/components/ScoreBoard/Aside.vue]: 用模板字符串更优雅

Comment (Artanic30) [reply src/student/components/ScoreBoard/Aside.vue]: 学习了

File: src/instructor/components/AddAssignment/Index.vue (hunk 2 above)
Comment (Artanic30) [reply src/instructor/components/AddAssignment/Index.vue]: save it to reconstitution

**Pull request merged**